README.md

```diff
@@ -1,2 +1,252 @@
-# devops-guidelines
-DevOps guidelines for any infrastructure as code project within Uniphar
+# Devops Guidelines
+
+The Uniphar Group targets the Azure cloud for most of the cloud workloads.
+The technical stack and tooling is around that of expected for an Azure
```

Comment:
Change the text to "and tooling is around that expected for an Azure", we are dropping the unneeded "of".